packages/react/src/components/ProtectedRoute/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import { Route, RouteProps } from 'react-router-dom';
+
+import { LogtoContext } from '../../context';
+
+const ProtectedRoute = ({ ...routeProperties }: RouteProps) => {
+  const { isAuthenticated, isLoading, signIn } = useContext(LogtoContext);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
```

Comment:
this should be customizable?

Reply:
Yes, we can allow passing a LoadingComponent through LogtoConfig, however, I'm not sure if we are going to do this in V1. Thoughts?

Reply:
imo `LogtoConfig` should have no loading components. actually `ProtectedRoute` should be implemented by the developer not us.

Reply:
Updated. PTAL